front/src/utils/getColorForShowId.ts

```diff
@@ -17,7 +17,7 @@ export const getUniqueColorsForShowIds = (showIds: number[]) => {
     while (findUniqueIndexStepsCount < colors.length) {
       // If the index isn't used yet, then choose this number and break loop
       if (!usedIndicies.hasOwnProperty(uniqueIndex)) {
-        console.log('uniqueIndex:', uniqueIndex);
+        //console.log('uniqueIndex:', uniqueIndex);
```

Comment:
I think I committed this by accident. You can delete this line if you get a chance

Reply:
this was interfering with test logs. I can delete it

Reply:
Sure, please delete 👍